xwiki-commons-core/xwiki-commons-properties/src/main/java/org/xwiki/properties/internal/DefaultPropertyDescriptor.java

```diff
@@ -91,6 +92,11 @@
      */
     private boolean advanced;
 
+    /**
+     * @see #getGroups()
+     */
+    private List<String> groups;
```

Comment:
Same comment than previously about `groups` vs `group`.

Instead of using directly a `List<String>` maybe it would be clearer to use some `PropertyGroup` (which contain the list) or a different name to not conflict with the annotation name.

Reply:
Ok I see.

Reply:
@tmortagne not sure about the name: `PropertyGroupContainer`.

Reply:
Definitely not `PropertyGroupContainer` IMO since it's IS a group. If `PropertyGroup` in a different package than the annotation then I guess `PropertyGroupDescriptor` is fine, consistent with other *Descriptor classes..